src/kernel/include/thd.h

```diff
@@ -472,17 +498,30 @@ curr_invstk_top(struct cos_cpu_local_info *cos_info)
 	return cos_info->invstk_top;
 }
 
+/* 
+ * This function aims to output the "in_fault" flag and return comp_info with the bit in captbl reset.
+ * Refer to the comments of comp_invstk_info to see more about this design.
```

Comment:
I don't think you need that comment on "refer to the comments...." for each of the function that is part of the fault-handling mechanism..

Reply:
sorry.. a nitpick again..